coopersystem-fsd/pcc-framework
# Add past course option and price field

## Description

- Additional event information fields will only be shown in the parent event

## Steps to test

1. Create an event and fill the fields;
2. See if fields are saved.

## Additional information
